roles/apim-gateway/templates/carbon-home/repository/conf/carbon.xml.j2

```diff
@@ -407,15 +407,15 @@
         -->
         <InternalKeyStore>
             <!-- Keystore file location-->
-            <Location>{{ internal_keystore_location }}</Location>
+            <Location>${carbon.home}/repository/resources/security/wso2carbon.jks</Location>
```

Comment:
Please do not remove the parameterization here. Keep it common for all the profiles as discussed.